addons/action/org.openhab.action.lgwebos/src/main/java/org/openhab/action/lgwebos/handler/AppActionHandler.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
```

Comment:
Please update all headers to 2018 e.g. by executing the `mvn license:format` at the root of the repository.